bin/create_yml.pl

```diff
@@ -88,8 +88,22 @@
         'hemato' => {
             'institute' => 'myeloid',
             'institute_owner' => 'myeloid',
+        },
+    },
+    'mody-cftr-aatv1-0' => {
+        'rankm' => 'SNV-RM-v5.0',
+        'svrankm' => 'SV-Panel-RM-v1.0',
+        'capture_kit' => 'mody-ctfr-aatv1-0',
+        'mody' => {
+            'institute' => 'mody-test',

```

Comment:
More for my understanding. I guess these "-test" are used now during testing, but not later, right?

Reply:
I kinda commented on it with my commit-message. But it's due to technical things with scout. There is no clinical institute for mody or cf yet. So I had to get gene-panel definitions from the validation institute, which Paul named -test, which is a weird name, should be called validation or similar